testing/scripts/test_rolling_updates.py

```diff
@@ -26,134 +30,6 @@ def to_resources_path(file_name):
 @pytest.mark.flaky
 @with_api_gateways
 class TestRollingHttp(object):
-    # Test updating a model with a new image version as the only change
```

Comment:
I understand correctly that these removed tests are equivalent to new addition bellow
```
        ("graph1.json", "graph2.json"),  # New image version
        ("graph1.json", "graph3.json"),  # New image version and new name of container
        ("graph1.json", "graph4.json"),  # New resource request but same image
        ("graph1.json", "graph5.json"),  # Update with multi-deployment new model
```
?

Reply:
That's correct. All of them had the same structure, so I thought it would be good to parametrise them.